app/src/main/java/org/thoughtcrime/securesms/contacts/SelectContactsToInviteToGroupActivity.kt

```diff
@@ -16,20 +16,22 @@ import com.bumptech.glide.Glide
 import dagger.hilt.android.AndroidEntryPoint
 
 @AndroidEntryPoint
-class SelectContactsActivity : ScreenLockActionBarActivity(), LoaderManager.LoaderCallbacks<List<String>> {
+class SelectContactsToInviteToGroupActivity : ScreenLockActionBarActivity(), LoaderManager.LoaderCallbacks<List<String>> {
     private lateinit var binding: ActivitySelectContactsBinding
+
     private var members = listOf<String>()
         set(value) { field = value; selectContactsAdapter.members = value }
+
     private lateinit var usersToExclude: Set<String>
 
     private val selectContactsAdapter by lazy {
         SelectContactsAdapter(this, Glide.with(this))
     }
 
     companion object {
-        val usersToExcludeKey = "usersToExcludeKey"
-        val emptyStateTextKey = "emptyStateTextKey"
-        val selectedContactsKey = "selectedContactsKey"
+        val USERS_TO_EXCLUDE_KEY  = "usersToExcludeKey"
+        val EMPTY_STATE_TEXT_KEY  = "emptyStateTextKey"
+        val SELECTED_CONTACTS_KEY = "selectedContactsKey"
```

Comment:
While we are at this maybe we can change them to const?

```suggestion
        const val USERS_TO_EXCLUDE_KEY  = "usersToExcludeKey"
        const val EMPTY_STATE_TEXT_KEY  = "emptyStateTextKey"
        const val SELECTED_CONTACTS_KEY = "selectedContactsKey"
```